public/app/features/explore/Logs/LogsMetaRow.tsx

```diff
@@ -68,9 +84,22 @@ export const LogsMetaRow = React.memo(
             type: 'application/json;charset=utf-8',
           });
 
-          const fileName = `Explore-logs-${dateTimeFormat(new Date())}.json`;
           saveAs(blob, fileName);
           break;
+        case DownloadFormat.CSV:
+          const dataFrame = logRows[0].dataFrame;
```

Comment:
We should process all dataFrames, not just the logRows[0].

Reply:
Should we download a separate file for each dataframe?